.pylintdict

```diff
@@ -353,10 +378,15 @@ ok
 oneee
 online
 onwards
+OperatorBase
+oplist
+oplist's
```

Comment:
Same here, since `oplist` is know this shouldn't be flagged as wrong word :slightly_smiling_face:

Reply:
If you remove 'oplist's' in the MacOS or Linux you get even with oplist already there:
************* Module aqua.operators.operator_combos.op_vec
qiskit/aqua/operators/operator_combos/op_vec.py:266:0: C0402: Wrong spelling of a word 'oplist's' in a docstring:
        OpVec's eval recursively evaluates each Operator in self.oplist's eval,
                                                                 ^^^^^^^^
Did you mean: ''op list's' or 'populist's' or 'oculist's' or 'violist's''? (wrong-spelling-in-docstring